test/ecoli_flux.jl

```diff
@@ -22,7 +22,7 @@ end
         sbmlfile * ".does.not.really.exist",
     )
 
-    @test mdl.units["mmol_per_gDW_per_hr"] == 2.7777e-7 * u"mol * g^-1 * s^-1"
+    @test mdl.units["mmol_per_gDW_per_hr"] ≈ 3.6001008028224795 * u"mol * g^-1 * s^-1"
```

Comment:
O_o are the float errors so bad?

Reply:
Tests actually pass on my computer with `==`, but don't want to rely on the full 17-digit representation for this to work on all systems. Also, this model is simply wrong 😬

Reply:
I'm scared but merging. :D